docs/joss-paper/paper.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'EvoMaster: A Search-Based System Test Generation Tool'
+tags:
+  - Java
+  - Kotlin
+  - SBST
+  - test generation
+  - system testing
+  - REST
+
+authors:
+  - name: Andrea Arcuri
+    orcid: 0000-0003-0799-2930
+    affiliation: 1
+  - name: Juan Pablo Galeotti
+    affiliation: 2
+  - name: Bogdan Marculescu
+    affiliation: 1
+  - name: Man Zhang
+    orcid: 0000-0003-1204-9322
+    affiliation: 1
+affiliations:
+ - name: Kristiania University College, Department of Technology, Oslo, Norway
+   index: 1
+ - name: FCEyN-UBA, and ICC, CONICET-UBA, Depto. de Computaci\'on, Buenos Aires, Argentina
+   index: 2
+date:  January 2019
+bibliography: paper.bib
+---
+
+
+# Summary
+
+Testing web/enterprise applications is complex and expensive when done manually.
+Therefore, in *Software Engineering* (SE) research, a lot of effort has been spent in trying 
+to design and implement novel techniques aimed at automating several different tasks in SE.
+*Search-Based Software Testing* (SBST) casts the problem of software testing as an optimization problem,
+aimed at for example maximizing code coverage and fault detection.   
```

Comment:
maybe wrap the "for example" between commas "aimed at, for example, maximizing"

Reply:
fixed